app/screens/ScorecardPreference/ScorecardPreference.js

```diff
@@ -41,7 +41,10 @@ class ScorecardPreference extends Component {
 
   loadProgramLanguage = async () => {
     const locales = getAllProgramLanguage(this.state.scorecard.program_id);
-    const languagesPickerFormat = locales.map((locale) => ({value: locale.code, label: locale.name}));
+    const languagesPickerFormat = locales.map((locale) => ({
+      value: locale.code,
+      label: locale.name == 'Khmer' ? 'ខ្មែរ' : locale.name,
```

Comment:
@limkimsan consider correcting in the datastore [{code: 'km', name: 'ខ្មែរ'}, {code: 'en', name: 'English'}] instead